sink/sink

```diff
@@ -412,17 +417,62 @@ def mkdir_and_chdir(base, identifier):
     os.fchmod(dirfd, 0o755)
     os.close(dirfd)
 
+def prune_logs(base_path = ".", dry_run = False):
+    reference_time = time.time()
+
+    entries = os.listdir(base_path)
+    for entry in entries:
+        path = os.path.join(base_path, entry)
+        log_path = os.path.join(path, "log")
+
+        # we only care about directories with our "log" file in them
+        if not os.path.isdir(path) or not os.path.exists(log_path):
+            continue
+
+        # keep release logs
```

Comment:
This is very Cockpit specific, and this sink is meant to be generic. Can we avoid having these lines? Is there another solution?